about.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Module1</title>
+    <link rel="stylesheet" href="about.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+<body>
+    <section class="nav-bar">
+        <div id="ham-menu">
+          <img
+            onclick="clickX()"
+            id="x"
+            src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+          />
+          <ul class="ham">
+            <li>
+              <a href="index.html" class="ham-items" onclick="clickX()"
+                >Home</a
+              >
+            </li>
+            <li>
+              <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+            </li>
+            <li>
+              <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+            </li>
+          </ul>
+        </div>
+        <header class="head flex-row">
+          <nav>
+            <img
+              onclick="clickMenu()"
+              id="menu"
+              src="./Images/Icon - Menu.png"
+              alt="Menu Logo"
+              class="menu-logo"
+            />
+            <a href="">
+              <img class="logo" src="Images/logo-black.png" alt="logo" />
+            </a>
+            <ul class="menu_items flex-row">
+              <li class="nav_items"><a href="index.html">Home</a></li>
+              <li class="nav_items"><a href="about.html">About</a></li>
+              <li class="nav_items"><a href="">Program</a></li>
+              <li class="nav_items"><a href="">Join</a></li>
+              <li class="nav_items"><a href="">Sponsor</a></li>
+            </ul>
+          </nav>
+        </header>
+        <div class="main-headline">
```

Comment:
- [ ] the first section of the about has a description text in a box. kindly add that
yours
![Screenshot (983)](https://user-images.githubusercontent.com/65298048/233437333-58875017-68a2-4d6a-b966-8ab8f12dd4f4.png)
original
![Screenshot (984)_LI](https://user-images.githubusercontent.com/65298048/233437490-6d985f75-0942-40e2-80bc-8e220d696f6b.jpg)